leocad.pro

```diff
@@ -9,7 +9,7 @@ greaterThan(QT_MAJOR_VERSION, 4) {
 INCLUDEPATH += qt common
 CONFIG += precompile_header incremental c++11
 win32 { 
-	PRECOMPILED_HEADER = lc_global.h
+	PRECOMPILED_HEADER = common/lc_global.h
```

Comment:
It's strange that this works for you, it doesn't compile for me unless I update lc_global.cpp also.